src/librustdoc/clean/mod.rs

```diff
@@ -1817,7 +1817,7 @@ pub(crate) fn clean_ty<'tcx>(ty: &hir::Ty<'tcx>, cx: &mut DocContext<'tcx>) -> T
                         // Only anon consts can implicitly capture params.
                         // FIXME: is this correct behavior?
                         let param_env = cx.tcx.param_env(*def_id);
-                        ct.normalize(cx.tcx, param_env)
+                        ct.normalize_internal(cx.tcx, param_env)
```

Comment:
This is a bit unfortunately necessary. I gues this could also be replaced with a call to `normalize_erasing_regions` below; I think I just forgot.

Reply:
Yeah I think that makes sense, anon consts under `min_const_generics` and `min_generic_const_args` are always trivial to evaluate like free aliases

Reply:
Actually I will replace it with normalize_erasing_regions for consistency.